def handleErr():
    try:
        """ if there  any error"""
    except:
        pass


